db/migrate/20241021110207_apply_soft_delete_for_course_user_and_associated_models.rb

```diff
@@ -0,0 +1,48 @@
+class ApplySoftDeleteForCourseUserAndAssociatedModels1 < ActiveRecord::Migration[7.2]
```

Comment:
`NameError: uninitialized constant ApplySoftDeleteForCourseUserAndAssociatedModels (NameError)`

Reply:
You get this error when run "rails db:migrate" right ?